esmvaltool/cmorizers/obs/cmor_config/CDS_UERRA.yml

```
@@ -0,0 +1,23 @@
+---
+
+# Common global attributes for Cmorizer output
+attributes:
+  frequency: 6h
+  dataset_id: CDS-UERRA
+  version: 'UERRA-HARMONIE'
+  tier: 3
+  modeling_realm: reanaly
+  project_id: OBS6
+  source: 'https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-uerra-europe-soil-levels?tab=form'
+  reference: 'cds-uerra'
+  comment: 'This dataset has been regridded for usage in ESMValTool'
+
+custom:
+  regrid: 0.25x0.25
+
+# Variables to cmorize
+variables:
+  sm:
+    mip: E6hr
+    raw: vsw
+    file: uerra_??????.nc
```

This is not a good choice if you want to add more variables in the future.
I would suggest to change it to:
```suggestion
    file: uerra_volumetric_soil_moisture_????.nc
```
which is similar to the name used in `ERA5` and `ERA-Interim`.